index.js

```diff
@@ -156,7 +156,8 @@ module.exports = {
     "property-no-vendor-prefix": true,
     "property-whitelist": null,
     "rule-empty-line-before": ["always", {
-      "except": ["first-nested"]
+      "except": ["first-nested"],
+      "ignoreAtRules": ["function", "if", "each", "include", "mixin", "for", "else", "import", "media"]
```

Comment:
I can't find `ignoreAtRules` in this rule's docs, I think it's `ignore`: https://stylelint.io/user-guide/rules/rule-empty-line-before

Reply:
Yup